questions.md

```diff
@@ -0,0 +1,5 @@
+# Questions on pull requests
+- What is the definition of the base branch
+- What is the definition of the head branch
+- How do I review the pull request and see changes
```

Comment:
You navigate to the repository, to pull requests, and start a review. You can make line-by-line changes, like I am doing now.